src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -1,18 +1,73 @@
 (ns status-im2.contexts.wallet.edit-account.view
   (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [quo.foundations.resources :as resources]
             [quo.theme :as quo.theme]
             [reagent.core :as reagent]
             [status-im2.contexts.wallet.common.screen-base.create-or-edit-account.view :as
              create-or-edit-account]
             [status-im2.contexts.wallet.edit-account.style :as style]
-            [utils.i18n :as i18n]))
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(def account-address "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045")
+
+(def mainnet
+  [{:title        "Mainnet"
+    :image        :icon-avatar
+    :image-props  {:icon (resources/get-network :ethereum)
+                   :size :size-20}
+    :action       :selector
+    :action-props {:type :checkbox}}])
+
+(def networks-list
+  [{:title        "Optimism"
+    :image        :icon-avatar
+    :image-props  {:icon (resources/get-network :optimism)
+                   :size :size-20}
+    :action       :selector
+    :action-props {:type :checkbox}}
+   {:title        "Arbitrum"
+    :image        :icon-avatar
+    :image-props  {:icon (resources/get-network :arbitrum)
+                   :size :size-20}
+    :action       :selector
+    :action-props {:type :checkbox}}])
+
+(defn network-preferences
```

Comment:
since this is used in multiple places in the designs can we move this to "src/status_im2/contexts/wallet/common/network-preferences/view.cljs"

also it would be great if this takes an `on-save` callback so that we can trigger this when the button is pressed. I think the use cases will be different on the different screens where it's used